app/lib/common/widget/FaqListItem.dart

```diff
@@ -0,0 +1,319 @@
+                                      fontWeight: FontWeight.w600,
+                                    ),
+                                  ),
+                                ),
+                                // new Spacer(),
+                                GestureDetector(
+                                  onTap: () => {
+                                    Fluttertoast.showToast(
+                                      msg: 'Bookmark icon tapped',
+                                      toastLength: Toast.LENGTH_SHORT,
+                                    ),
+                                  },
+                                  child: Image.asset(
+                                    'assets/images/bookmark.png',
+                                  ),
+                                )
+                              ],
+                            ),
+                          ),
+                          Padding(
+                            padding: EdgeInsets.only(top: 12.0, bottom: 12.0),
+                            child: Row(
+                              crossAxisAlignment: CrossAxisAlignment.start,
+                              mainAxisAlignment: MainAxisAlignment.center,
+                              children: [
+                                Container(
+                                  height: 40.0,
+                                  padding: EdgeInsets.all(8.0),
+                                  margin: EdgeInsets.only(left: 12.0),
+                                  decoration: BoxDecoration(
+                                    borderRadius: BorderRadius.circular(10),
+                                    border: Border.all(
+                                      color: Color(0XFFFFFFFFF),
+                                    ),
+                                  ),
+                                  child: Row(
+                                    crossAxisAlignment:
+                                        CrossAxisAlignment.start,
+                                    children: [
+                                      Image.asset(
+                                        'assets/images/asakerImage.png',
+                                      ),
+                                      Padding(
+                                        padding: const EdgeInsets.only(
+                                          left: 8.0,
+                                        ),
+                                        child: Text(
+                                          'Support',
+                                          style: GoogleFonts.montserrat(
+                                            color: Colors.white,
+                                            fontSize: 16,
+                                          ),
+                                        ),
+                                      )
+                                    ],
+                                  ),
```

Comment:
This PR introduced the idea of mocking the data right from the native side, as it makes it easier to replace it with the actual code later. Please let's not re-introduce mocked data right in the frontend (as this often requires significant changes when using actual async-function calls to the SDK). Let's mock this native side instead.